scripts/core/driver.yml

```diff
@@ -271,3 +271,26 @@ members:
     - type: $x_driver_handle_t
       name: handle
       desc: "[in] handle of the driver instance"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Retrieves a message associated with the returned code of the last call made."
+version: "1.6"
+class: $xDriver
+name: GetLastResultString
+details:
+    - "String returned is thread local."
+    - |
+       String may be empty if last call returned $X_RESULT_SUCCESS or if driver
+       determines that error code is already explicit enough to convey error cause.
+    - "Memory pointed by ppString is owned by driver."
+    - "String returned is null-terminated."
+params:
+    - type: $x_driver_handle_t
+      name: hDriver
+      desc: "[in] handle of the driver instance"
+    - type: "const char**"
+      name: ppString
+      desc: "[in,out] pointer to an array of characters containing message associated with the last call."
+    - type: "size_t*"
+      name: pSize
```

Comment:
`pSize` is unnecessary because string is null-terminated

Reply:
@bmyates . right, I was thinking on that. Users could just use strlen.